### PR TITLE
Add support for Electronut labs bluey board

### PR DESCRIPTION
**Pull request by ntavish**

### CODE REVIEW

File: variants/bluey/variant.h [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+#include "WVariant.h"
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif // __cplusplus
+
+// Number of pins defined in PinDescription array
+// NOTE: NUM_DIGITAL_PINS does not seem to be used anywhere,
```
Comment (sandeepmistry) [inline variants/bluey/variant.h]: It's an Arduino core thing, you should be able to loop from 0 to `NUM_DIGITAL_PINS` and output LOW or HIGH and make something external change.

Please remove this comment once you understand and have the right value for it.

File: variants/bluey/variant.h [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+static const uint8_t A0  = PIN_A0 ; // AIN0 / AREF / P0.02
+static const uint8_t A1  = PIN_A1 ; // AIN1
+static const uint8_t A2  = PIN_A2 ; // AIN2
+static const uint8_t A3  = PIN_A3 ; // AIN4
+static const uint8_t A4  = PIN_A4 ; // AIN5
+static const uint8_t A5  = PIN_A5 ; // AIN6
+static const uint8_t A6  = PIN_A6 ; // AIN7
+#define ADC_RESOLUTION    10
+
+// XXX check ADC resolution, doesn't seem to be in use
```
Comment (sandeepmistry) [inline variants/bluey/variant.h]: Again, another standard Arduino constant. Please remove the comment.

**Comment by ntavish** [comment]
removed those comments, please see if this is okay now

File: variants/bluey/variant.h [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  You should have received a copy of the GNU Lesser General Public
+  License along with this library; if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+*/
+
+#ifndef _VARIANT_NRF52_DK_
+#define _VARIANT_NRF52_DK_
+
+/** Master clock frequency */
+// TODO confirm
```
Comment (sandeepmistry) [inline variants/bluey/variant.h]: Is this confirmed?

**Comment by sandeepmistry** [comment]
@ntavish could you also please add an entry to the [.travis.yml](https://github.com/sandeepmistry/arduino-nRF5/blob/master/.travis.yml).

File: docs/package_nRF5_boards_index.json [hunk 4]
```diff
@@ -98,7 +98,8 @@
             {"name": "RedBear Blend 2"},
             {"name": "RedBearLab BLE Nano"},
             {"name": "RedBearLab nRF51822"},
-            {"name": "Waveshare BLE400"}
+            {"name": "Waveshare BLE400"},
+            {"name": "Electronut Labs Bluey"}
```
Comment (sandeepmistry) [inline docs/package_nRF5_boards_index.json]: Please remove this line, it will be added when the changes for the board are part of a release. Thanks.

**Comment by ntavish** [comment]
Done changes as suggested.

**Pull request merged**